### PR TITLE
Docs: updating GKE and AKS Kubernetes versions

### PR DESCRIPTION
**Pull request by aLekSer**
Kubernetes version: GKE website changes, AKS website and Terraform
changes.

**What type of PR is this?**

/kind documentation

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:
Relates to #1649

**Special notes for your reviewer**:
Merging 2 small PRs into one.

### CODE REVIEW

File: site/content/en/docs/Installation/Creating Cluster/aks.md [hunk 1]
```diff
@@ -27,8 +27,15 @@ AKS_LOCATION=westeurope          # Azure region in which you'll deploy your AKS
 az group create --name $AKS_RESOURCE_GROUP --location $AKS_LOCATION
 
 # Create the AKS cluster - this might take some time. Type 'az aks create -h' to see all available options
+
+{{% feature expiryVersion="1.9.0" %}}
```
Comment (roberthbailey) [inline site/content/en/docs/Installation/Creating Cluster/aks.md:31]: Can you double check how this renders? Last time I tried this I don't think it worked to insert these inside of the triple backticks.

File: install/terraform/modules/aks/aks.tf [hunk 2]
```diff
@@ -60,7 +60,7 @@ resource "azurerm_kubernetes_cluster" "agones" {
   resource_group_name = azurerm_resource_group.agones_rg.name
   dns_prefix          = "agones"
 
-  kubernetes_version = "1.15.10"
+  kubernetes_version = "1.16.10"
```
Comment (roberthbailey) [inline install/terraform/modules/aks/aks.tf:63]: Is 1.16.10 the latest version of 1.16 on aks?

**Comment by markmandel** [comment]
Updating, so we can grab a render.

File: site/content/en/docs/Installation/Creating Cluster/aks.md (hunk 1 above)
Comment (roberthbailey) [reply site/content/en/docs/Installation/Creating Cluster/aks.md:31]: It renders a giant blank space inside of the gray box: https://9723542-dot-preview-dot-agones-images.appspot.com/site/docs/installation/creating-cluster/aks/

File: install/terraform/modules/aks/aks.tf (hunk 2 above)
Comment (aLekSer) [reply install/terraform/modules/aks/aks.tf:63]: According to the docs, yes: https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions
I am going to renew my subscription to AKS and make a fast test.
Also this command would be useful:
```
az aks get-versions --location eastus --output table
```

File: site/content/en/docs/Installation/Creating Cluster/aks.md (hunk 1 above)
Comment (aLekSer) [reply site/content/en/docs/Installation/Creating Cluster/aks.md:31]: Yes, let me fix the render.

File: install/terraform/modules/aks/aks.tf (hunk 2 above)
Comment (aLekSer) [reply install/terraform/modules/aks/aks.tf]: Renewed my subscription. Testing EKS cluster with Terraform now.
```
az aks get-versions --location eastus --output table
KubernetesVersion    Upgrades
-------------------  -------------------------
1.17.9               None available
1.17.7               1.17.9
1.16.13              1.17.7, 1.17.9
```

Comment (roberthbailey) [reply install/terraform/modules/aks/aks.tf]: Thanks for double checking.

**Review by roberthbailey — APPROVED** [review]

**Pull request merged**